docs/3_user_guide.rst

```diff
@@ -43,13 +43,9 @@ subtracted from the total cases to obtain the *in control*
 time series, i.e. the time series without outbreaks.
 
 If you do not have any labeled outbreak data, but just the raw
-counts you can fill the ``n_outbreak_cases`` column with
-zeros under the assumption that your data is in fact
-*in control* data.
-
-.. code-block:: python
-
-    data["n_outbreak_cases"] = 0
+counts ``epysurv`` implicitly adds the ``n_outbreak_cases`` column filled
+with zeros under the assumption that your data is in fact
```

Comment:
I thought that NaNs would be better since zeros suggest that we "know" that there are not outbreak cases but maybe we just don't know

Reply:
Doing it with NaNs is not possible, as you could not subtract them from the `n_cases` column. What happens is that simply nothing is subtracted from the `n_cases` column. But maybe the wording is a still a bit confusing.